### PR TITLE
GM: modified worker mount mode

### PR DESCRIPTION
**Pull request by JimmyYang20**
Signed-off-by: JimmyYang20 <yangjin39@huawei.com>

### CODE REVIEW

File: pkg/globalmanager/lifelonglearningjob.go [hunk 1]
```diff
@@ -675,6 +675,7 @@ func (jc *LifelongLearningJobController) createInferPod(job *sednav1.LifelongLea
 			URL: &MountURL{
 				URL:    inferModelURL,
 				Secret: jobSecret,
+				Mode:   workerMountWriteOnly,
```
Comment (llhuii) [inline pkg/globalmanager/lifelonglearningjob.go:678]: Why does inferModel need to be writable?

Comment (JimmyYang20) [reply pkg/globalmanager/lifelonglearningjob.go]: define new mode: workerMountDownloadOnly

**Pull request merged**